internal/constants/constants.go

```diff
@@ -105,4 +106,6 @@ const (
 	UkiSysrootDir          = "sysroot"
 	PersistentStateTarget  = "/usr/local/.state"
 	LogDir                 = "/run/immucore"
+	PathAppend             = "/usr/bin:/usr/sbin:/bin:/sbin"
+	PathEnvKey             = "PATH"
```

Comment:
Well, it's a key=val format no? So I thought key made kind of sense. Tbh I struggled with a decent name to convey what this is lol

Reply:
I see what you mean, I would say for me is confusing because the name already has "path" which is what identifies the environment variable, so while it is correct that it is the key that references that env var, the "key" name makes me wonder if it's a traditional env var or something else. But anyway not a strong requirement from my side.

Reply:
Maybe I can just name It PATH 🤣